pkg/process/procutil/data_scrubber_fallback_test.go

```diff
@@ -0,0 +1,61 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build !windows
+
+package procutil
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStripArguments(t *testing.T) {
+	for _, tc := range []struct {
+		name     string
+		cmdline  []string
+		expected string
+	}{
+		{
+			name:     "OS parse",
+			cmdline:  []string{"agent", "-password", "1234"},
```

Comment:
What happens to a cmdline like this?
```
[]string{"agent", "-password", "1234", "--config", "abcd" },
```

Reply:
Actually, nvm. Is this `stripArguments` supposed to remove all arguments leaving the command that's supposed to run? we should add a comment to the function to highlight that.

Reply:
In this case, since it is a non windows test, it will use the stripArguments function from data_scrubber_fallback.go which returns the first token if cmdline has more than one element. So it will return "agent" as well.

Reply:
Oh oops - didn't see previous comment. Will add a comment to the function.